test.cpp

```
@@ -0,0 +1,132 @@
+
+void searchStudent(Student* students, int count, string name) {
+    for (int i = 0; i < count; i++) {
+        if (students[i].name = name) {
+            cout << "Student found: " << students[i].name << "\n";
+            return;
+        }
+    }
+    cout << "Student not found\n";
+}
+
+void deleteStudent(Student* students, int& count, string name) {
```

Cppcheck (2.10-2)

[performance] Function parameter 'name' should be passed by const reference.

(passedByValue)